superchic.sh

```diff
@@ -0,0 +1,59 @@
+package: SuperCHIC
+version: "%(tag_basename)s%(defaults_upper)s"
+tag: "alice/v2.04"
+source: https://github.com/hcab14/SuperCHIC.git
+requires:
+ - lhapdf
+build_requires:
+ - curl
+ - "GCC-Toolchain:(?!osx)"
+---
+#!/bin/bash -e
+
+rsync -a --exclude '**/.git' $SOURCEDIR/ ./
+
+patch makefile <<EOF
+8c8
+< LHAPDFLIB = /usr/local/Cellar/lhapdf/6.1.5/lib
+---
+> LHAPDFLIB = ${LHAPDF_ROOT}/lib
+
+EOF
+
+mkdir -p obj
+
+make ${JOBS+-j $JOBS}
+
+# the makefile does not contain "install" -> manual installation
+cp -R bin $INSTALLROOT
+
+# install lhapdf sets MMHT2014*
+lhapdf update
```

Comment:
How can this possibly work? The LHAPDF packages was built and installed before, there's no way to add PDFsets at this stage. This has to happen when building the LHAPDF (or the PDFsets, but the separation of the binaries and the PDFsets is still on the todo list).